### DISCUSSION TITLE
Wion m3u8s

### DISCUSSION THREAD

**Discussion by darrencycheung** [iptv-org/community#87] [Q&A] [answered]
Anyone who have Wion m3u8?

**Comment by TKList** [comment]
This one? 
https://www.youtube.com/embed/KyUT37gNm5w

**Reply by darrencycheung** [reply]
Yes. But what about normal m3u8? https://dai.google.com/linear/hls/pb/event/x_dFvsORSZWERbxrltEk0g/stream/9ee9f771-6b1f-4cc1-958a-913814fde60d:SIN2/master.m3u8

**Comment by TKList** [comment, marked as answer]
https://dai.google.com/linear/hls/event/x_dFvsORSZWERbxrltEk0g/master.m3u8

**Reply by darrencycheung** [reply]
Thanks. I hope that link will work for a long time.

**Reply by fotvnetwork** [reply]
Stable link from dai.google
